docs/enterprise.md

```diff
@@ -20,16 +20,16 @@ and [the license](https://github.com/VictoriaMetrics/VictoriaMetrics/blob/master
 
 The use of VictoriaMetrics enterprise components is permitted in the following cases:
 
-- Evaluation use in non-production setups. Just download and run enterprise binaries or packages of VictoriaMetrics
-  components from usual places - [releases page](https://github.com/VictoriaMetrics/VictoriaMetrics/releases) and [docker hub](https://hub.docker.com/u/victoriametrics).
+- Evaluation use in non-production setups. Please, request trial license [here](https://victoriametrics.com/products/enterprise/).
```

Comment:
we need to add something like:

if you want to try the enterprise features you can request free trial <link>, we email you license key after submitting the form